File: problems/baekjoon/p1330.cpp
/*
 * (210718) 두 수 비교하기
 * https://www.acmicpc.net/problem/1330
 */

#include <cstdio>

int main() {
	int A, B;
	scanf("%d %d", &A, &B);

	if (A > B) {
		printf(">\n");
	} else if (A < B) {
		printf("<\n");
	} else {
		printf("==\n");
	}

	return 0;
}

File: problems/csharp_baekjoon/p1330.cs
/*
 * (220413) 두 수 비교하기
 * https://www.acmicpc.net/problem/1330
 */

using System;

namespace Baekjoon
{
  class Program
  {
    static void Main()
    {
      int [] nums = Array.ConvertAll(Console.ReadLine().Split(), int.Parse);

      if (nums[0] > nums[1]) Console.WriteLine(">");
      else if (nums[0] < nums[1]) Console.WriteLine("<");
      else Console.WriteLine("==");
    }
  }
}
